customcommands/bot.go

```diff
@@ -1088,8 +1088,13 @@ func ExecuteCustomCommand(cmd *models.CustomCommand, tmplCtx *templates.Context)
 	lockHandle := CCExecLock.Lock(lockKey, time.Minute, time.Minute*10)
 	if lockHandle == -1 {
 		f.Warn("Exceeded max lock attempts for cc")
+		chn := tmplCtx.CurrentFrame.CS.ID
```

Comment:
I definitely don’t like `chn` as the variable name, both because it breaks yag’s convention for abbreviation of channel variable names and because it is not immediately clear which channel this variable contains. Have you considered `errChannel`?

Reply:
Oh my borBot brain didn't think about that

Reply:
Changed it :3